http-server-netty/src/main/java/io/micronaut/http/server/netty/configuration/NettyHttpServerConfiguration.java

```diff
@@ -1071,7 +1071,7 @@ public static class Parent extends EventLoopConfig {
     }
 
     /**
-     * Allows configuration of properties for the {@link io.micronaut.http.server.netty.types.files.FileTypeHandler}.
+     * Allows configuration of properties for the {@link io.micronaut.http.server.netty.body.AbstractFileBodyWriter}.
```

Comment:
@yawkat is this change of javadoc correct?

Reply:
@yawkat Hi! Could you answer? I think, this is last unanswered question

Reply:
yes